packages/amplify-appsync-simulator/src/velocity/util/info.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphQLResolveInfo, SelectionNode } from 'graphql';
+import { print } from 'graphql/language/printer';
+
+const getSelectionSet = (nodes: readonly SelectionNode[], prefix: string = null) => {
```

Comment:
Could you add some tests to test cases where prefix is passed. Also could you add support for aliases

Reply:
@yuth Support for alises added. 
test for prefix is included in test.